packages/parcel-transformer/src/transformer.tsx

```diff
@@ -111,13 +115,14 @@ export default new Transformer<UserlandOpts>({
     return [asset];
   },
 
-  generate({ ast, asset }) {
+  async generate({ ast, asset }) {
     // TODO: We're using babels standard generator. Internally parcel does some hacks in
     // the official Babel transformer to make it faster - using ASTree directly.
     // Perhaps we should do the same thing.
-    const { code, map } = generate(ast.program, {
+    const { code } = generate(ast.program, {
       filename: asset.filePath,
     });
+    const map = await asset.getMap();
```

Comment:
This will create inaccurate source maps, as it will use an old one. You should be able to update the line before to:

```
const { code, map } = generate(ast.program, {
  filename: asset.filePath,
  sourceMaps: true,
});
```